app/config/services.yml

```diff
@@ -5,6 +5,12 @@ services:
     cache.null_provider:
         class: Symfony\Component\Cache\Adapter\NullAdapter
 
+    cache.bbc:
```

Comment:
cache.* is the namespace that is created when you [configure the framework cache bits](http://symfony.com/doc/current/reference/configuration/framework.html#reference-cache). I don't think we should put this in there.

At the bottom of this file we define `pps.mapper_factory` and `pps.service_factory` as the utils that we need to setup the pages services. How about defining `pps.cache` down there too?